source/MRViewer/ImGuiMenu.cpp

```diff
@@ -1162,6 +1163,27 @@ float ImGuiMenu::drawSelectionInformation_()
                 pObj->setName( renameBuffer_ );
                 lastRenameObj_.reset();
             }
+
+            if ( auto pObjLabel = std::dynamic_pointer_cast<ObjectLabel>( pObj ) )
+            {
+                if ( pObjLabel != oldLabelName_.obj )
+                {
+                    oldLabelName_.obj = pObjLabel;
+                    const auto positionedText = pObjLabel->getLabel();
+                    oldLabelName_.lastLabel = positionedText.text;
+                    oldLabelName_.labelBuffer = oldLabelName_.lastLabel;
```

Comment:
why not `const auto& positionedText = pObjLabel->getLabel();`

Reply:
please rename field `oldLabelName_` it is not name and does not affect lables name